charts/plex-media-server/templates/statefulset.yaml

```
@@ -100,8 +100,8 @@ spec:
           value: {{ .Values.ingress.url }}:443
         {{- end }}
         {{- range $key, $value := .Values.extraEnv }}
-        - name: {{ $key }}
-          value: {{ $value }}
+        - name: "{{ $key }}"
+          value: "{{ $value }}"
```

nit: I think we can skip quoting `key` here. My preference would be to pipe the value to the `quote` function like we do on the rest of the template, for consistency (see `priorityClassName` in L30 above)

```suggestion
        - name: {{ $key }}
          value: {{ $value | quote }}
```

---

I believe I have address all of your suggestions and pushed the changes, thank you for the feedback!